### PR TITLE
Update treasuredata.md

### PR DESCRIPTION
**Pull request by alex-braze**
Adding a new section to the Cohort Import docs

# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ Search for the Braze Integration in the **Catalog**, then hover over the icon an
 
 ![Treasure Data Integrations Hub Catalog]({% image_buster /assets/img/treasure_data/cohort/cohort2.png %}) 
 
+### Step 3: Define your cohort Audience
+
+Cohorts can be synced to Braze through an **Activation** in the **Audience Studio** or through executing a **Query** in the **Data Workbench**
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:45]:
```suggestion
Cohorts can be synced to Braze through an **Activation** in the **Audience Studio** or through executing a **Query** in the **Data Workbench**.
```

Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:45]:
```suggestion
Sync your cohorts to Braze through an activation in the **Audience Studio** or by executing a query in the **Data Workbench**.
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 2]
```diff
@@ -40,6 +40,16 @@ Search for the Braze Integration in the **Catalog**, then hover over the icon an
...
+
+{% alert important %}
+Only users who already exist within Braze will be added or removed from a Cohort. Cohort Import **will not** create new users in Braze.
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:48]:
```suggestion
Only users who already exist within Braze will be added or removed from a cohort. Cohort Import will not create new users in Braze.
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 3]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
 
 ![Treasure Data Integrations Hub Catalog]({% image_buster /assets/img/treasure_data/cohort/cohort5.png %}) 
 
+
+| Export Result Mapping |	Description	| 
+| ----------- | ----------- |
+| Cohort ID	| This is the backend cohort identifier that will be sent to Braze. 	|
+| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in Braze's Segmentation tool. If this is not set, the **Cohort ID** will be used as the **Cohort Name**	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:168]:
```suggestion
| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in the Braze Segmentation tool. If this is not set, the `Cohort ID` will be used as the `Cohort Name`.	|
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 4]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+| Operation	| Used to determine whether the query should add or remove profiles from the Cohort in Braze	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:169]:
```suggestion
| Operation	| Used to determine whether the query should add or remove profiles from the cohort in Braze.	| 
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 5]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
 
 ![Treasure Data Integrations Hub Catalog]({% image_buster /assets/img/treasure_data/cohort/cohort5.png %}) 
 
+
+| Export Result Mapping |	Description	| 
+| ----------- | ----------- |
+| Cohort ID	| This is the backend cohort identifier that will be sent to Braze. 	|
+| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in Braze's Segmentation tool. If this is not set, the **Cohort ID** will be used as the **Cohort Name**	|
+| Operation	| Used to determine whether the query should add or remove profiles from the Cohort in Braze	| 
+| Aliases (Optional) | When defined, the name of the corresponding column within your Query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:170]:
```suggestion
| Aliases (Optional) | When defined, the name of the corresponding column within your Query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`	| 
```
```suggestion
| Aliases (Optional) | When defined, the name of the corresponding column within your query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`.	| 
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 6]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
 ![Treasure Data Integrations Hub Catalog]({% image_buster /assets/img/treasure_data/cohort/cohort5.png %}) 
 
+
+| Export Result Mapping |	Description	| 
+| ----------- | ----------- |
+| Cohort ID	| This is the backend cohort identifier that will be sent to Braze. 	|
+| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in Braze's Segmentation tool. If this is not set, the **Cohort ID** will be used as the **Cohort Name**	|
+| Operation	| Used to determine whether the query should add or remove profiles from the Cohort in Braze	| 
+| Aliases (Optional) | When defined, the name of the corresponding column within your Query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`	| 
+| Thread Count | Number of concurrent API calls |
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:171]:
```suggestion
| Thread Count | Number of concurrent API calls. |
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 7]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+
+Create a new Segment or choose an existing Segment to sync to Braze as a Cohort. Within the Segment, choose **Create activation**
+
+#### Activation Details
+
+![Treasure Data Integrations Activation Details]({% image_buster /assets/img/treasure_data/cohort/cohort7.png %}) 
+
+| Activation Detail Setting |	Description	| 
+| ----------- | ----------- |
+| Activation Name	| The name of your Activation	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:191]:
```suggestion
| Activation Name	| The name of your activation.	|
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 8]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+| Activation Description| A brief description of the Activation	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:192]:
```suggestion
| Activation Description| A brief description of the activation.	|
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 9]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+| Authentication	| Choose the Braze Cohort authentication created in Step 2	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:193]:
```suggestion
| Authentication	| Choose the Braze cohort authentication created in step 2.	| 
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 10]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+
+![Treasure Data Integrations Activation Details]({% image_buster /assets/img/treasure_data/cohort/cohort7.png %}) 
+
+| Activation Detail Setting |	Description	| 
+| ----------- | ----------- |
+| Activation Name	| The name of your Activation	|
+| Activation Description| A brief description of the Activation	|
+| Authentication	| Choose the Braze Cohort authentication created in Step 2	| 
+| Cohort ID	| This is the backend cohort identifier that will be sent to Braze. 	|
+| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in Braze's Segmentation tool. If this is not set, the **Cohort ID** will be used as the **Cohort Name**	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:195]:
```suggestion
| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in the Braze Segmentation tool. If this is not set, the `Cohort ID` will be used as the `Cohort Name`.	|
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 11]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+| Operation	| Used to determine whether the query should add or remove profiles from the Cohort in Braze	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:196]:
```suggestion
| Operation	| Used to determine whether the query should add or remove profiles from the cohort in Braze.	| 
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 12]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+
+| Activation Detail Setting |	Description	| 
+| ----------- | ----------- |
+| Activation Name	| The name of your Activation	|
+| Activation Description| A brief description of the Activation	|
+| Authentication	| Choose the Braze Cohort authentication created in Step 2	| 
+| Cohort ID	| This is the backend cohort identifier that will be sent to Braze. 	|
+| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in Braze's Segmentation tool. If this is not set, the **Cohort ID** will be used as the **Cohort Name**	|
+| Operation	| Used to determine whether the query should add or remove profiles from the Cohort in Braze	| 
+| Aliases (Optional) | When defined, the name of the corresponding column within your Query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:197]:
```suggestion
| Aliases (Optional) | When defined, the name of the corresponding column within your query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`.	| 
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 13]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+| Activation Detail Setting |	Description	| 
+| ----------- | ----------- |
+| Activation Name	| The name of your Activation	|
+| Activation Description| A brief description of the Activation	|
+| Authentication	| Choose the Braze Cohort authentication created in Step 2	| 
+| Cohort ID	| This is the backend cohort identifier that will be sent to Braze. 	|
+| Cohort Name (Optional)	| This is the name that will appear within the Cohort Filter in Braze's Segmentation tool. If this is not set, the **Cohort ID** will be used as the **Cohort Name**	|
+| Operation	| Used to determine whether the query should add or remove profiles from the Cohort in Braze	| 
+| Aliases (Optional) | When defined, the name of the corresponding column within your Query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`	| 
+| Thread Count | Number of concurrent API calls |
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:198]:
```suggestion
| Thread Count | Number of concurrent API calls. |
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 14]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+
+
+#### Activation Output Mapping
+
+![Treasure Data Integrations Activation Output Mapping]({% image_buster /assets/img/treasure_data/cohort/cohort6.png %}) 
+
+| Activation Output Mapping |	Description	| 
+| ----------- | ----------- |
+| Attribute Columns	| Determine the Columns from your Segment Database that will be mapped as Identifiers when syncing profiles to a Braze Cohort.	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:207]:
```suggestion
| Attribute Columns	| Determine the columns from your Segment database that will be mapped as identifiers when syncing profiles to a Braze cohort.	|
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 15]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+| String Builder| The String Builder is not necessary for this integration.	|
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:208]:
```suggestion
| String Builder| The string builder is not necessary for the Braze integration.	|
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 16]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+
+{% alert important %}
+ - When using `device_id` as the identifier the **Output Column Name** must be named `device_ids`
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:211]:
```suggestion
 - When using `device_id` as the identifier, the **Output Column Name** must be named `device_ids`.
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 17]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+ - When using aliases as the identifier, the **Output Column Name** must be the name of the corresponding column within your Query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:212]:
```suggestion
 - When using aliases as the identifier, the **Output Column Name** must be the name of the corresponding column within your query will be sent as the `alias_label`, and the values of each row in the column will be sent as the `alias_name`.
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md [hunk 18]
```diff
@@ -151,12 +161,69 @@ Once the query has been built, select **Export Results**. You can select an exis
...
+ - When using `external_id` as the identifier the **Output Column Name** must be named `user_ids`
+{% endalert %}
+
+All non-relevant or misnamed column names will be ignored. You may choose to use more than one identifier in your syncs.
+
+#### Activation Schedule
+
+Define your desired sync schedule and save your Activation.
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/treasuredata.md:220]:
```suggestion
Define your desired sync schedule and save your activation.
```

**Comment by josh-mccrowell-braze** [comment]
Hey @internetisaiah! I went through and provided a light round of style edits. I ran out of time, however, to finish this task. I notice that the tabbing is wonky on this page. Can you fix that and provide any other fixes this branch needs? Thanks!

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**